onset_maproom/cess.py

```diff
@@ -0,0 +1,117 @@
+#temporary file for cessation date functions; 
+#these will be moved to the main calc.py file when finished
```

Comment:
Remove this file now?

Reply:
I was going to wait for Remi to look over the changes I made the the functions before putting them in the main calc.py file if that's okay. The functions in calc.py are placeholders just for getting the plots working

Reply:
I've looked only at calc.py recently. Maybe remove that one if now irrelevant and to avoid confusion or duplicating review?